### PR TITLE
Add "Para Que Serve" section

### LINKED ISSUE #3: [landing page] create "para que serve" section
https://www.figma.com/file/BfK7HtPsvxZAwZLg1ebz2u/Tracovid?node-id=1193%3A158

### PR DESCRIPTION
**Pull request by taniaprocha**

### CODE REVIEW

File: src/sections/para-que-serve/index.module.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+.subtitle {
+  max-width: 676px;
+  margin: 21px auto 0 auto;
+}
+
+.content {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 60px 0;
```
Comment (ruisaraiva19) [inline src/sections/para-que-serve/index.module.css]: This padding is not needed.

File: src/sections/para-que-serve/index.module.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+.subtitle {
+  max-width: 676px;
+  margin: 21px auto 0 auto;
+}
+
+.content {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
```
Comment (ruisaraiva19) [inline src/sections/para-que-serve/index.module.css:27]: We also need to align items vertically to the center and `max-height: 317px` (only desktop).
```css
align-items: center;
```

File: src/sections/para-que-serve/index.module.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
+@import "../../styles/all.module.css";
+
+.root {
+  max-width: 1220px;
+  padding: 74px 0;
+  margin: 0 auto;
+  text-align: center;
+
```
Comment (ruisaraiva19) [inline src/sections/para-que-serve/index.module.css:8]: Please remove this extra empty line.

File: src/sections/para-que-serve/index.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+            ajudar o país a recuperar do impacto desta pandemia.
+          </Typography>
+        </div>
+        <div className={styles.column}>
+          <Typography color="gray" variant="mediumBody">
+            É ainda permitido aos utilizadores da aplicação acompanhar, em tempo
+            real, o impacto da pandameia na zona onde se encontram, com base nas
+            actualizações enviadas por todos os utilizadores. A privacidade dos
+            dados recolhidos é tratada com o máximo cuidado. Leia os nossos
+            Termos & Condições.
```
Comment (ruisaraiva19) [inline src/sections/para-que-serve/index.jsx:38]: The `Termos & Condições` should be a link.

**Review by ruisaraiva19 — CHANGES_REQUESTED** [review]

**Review by ruisaraiva19 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**